user/test/com/google/gwt/emultest/java/math/BigIntegerConstructorsTest.java

```diff
@@ -605,6 +606,62 @@ public void testConstructorSignBytesZeroNull3() {
     assertEquals("incorrect sign", 0, aNumber.signum());
   }
 
+  @Test
+  public void testConstructorWithZeroBytes_Offset_Length() {
+    byte[] aByte = {};
+    try {
+      new BigInteger(aByte, 2, 4);
+      fail("NumberFormatException has not been caught");
+    } catch (NumberFormatException e) {
+    }
+  }
+
+  @Test
+  public void testConstructorBytes_Offset_ZeroLength() {
+    byte[] aByte = {0, 0, 12, 34, 56, 78, 0, 0};
```

Comment:
Just keep in mind that if the implementation was wrong and it was reading the 0th or 1st position, the value would still be zero... might be better to replace the zeros here with something non-zero.